packages/react-query/scripts/utils.js

```diff
@@ -0,0 +1,40 @@
+import fs from 'fs'
+import path, { dirname } from 'path'
+import { fileURLToPath } from 'url'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = dirname(__filename)
+
+const dir = path.resolve(__dirname, '..', 'dist')
+
+export function loadModule(name) {
+  try {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-argument
+    return require(name)
```

Comment:
Even though this is an ES Module, I use `require` because it is faster and simpler than `import`.

- **Fast and Simple Operation**

`require` works synchronously, making it quick and straightforward to load modules. In contrast, `import` must work asynchronously, requiring the use of async/await syntax, which increases code complexity.

-  **Stability**

Importing JSON in the ES module environment of Node.js is still experimental. For example, to import JSON files in ES modules, need to use the `--experimental-json-modules` flag, which adds extra hassle and displays warning messages at runtime.